workspaces/client.go

```diff
@@ -0,0 +1,43 @@
+package workspaces
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type WorkspaceClient struct {
+	URL    string
+	client *http.Client
+	logger logrus.FieldLogger
+}
+
+// NewHTTPClient creates a workspace HTTP client for managing workspace
+func NewHTTPClient(URL string, client *http.Client) *WorkspaceClient {
+	return &WorkspaceClient{
+		URL:    URL,
+		client: client,
+	}
+}
+
+// DeleteWorkspace makes an http request to delete a workspace
+func (c *WorkspaceClient) DeleteWorkspace(installationID string) error {
+	if installationID == "" {
+		return errors.New("failed to delete workspace, installationID is required")
+	}
+	endpoint := fmt.Sprintf("%s/installation/%s", c.URL, installationID)
```

Comment:
👍🏾

Reply:
@spirosoik does it handle the customer web server deletion too?

Reply:
yes actually this is the part here.